pkg/volume/vsphere_volume/attacher.go

```diff
@@ -205,12 +206,17 @@ func (plugin *vsphereVolumePlugin) GetDeviceMountRefs(deviceMountPath string) ([
 
 // MountDevice mounts device to global mount point.
 func (attacher *vsphereVMDKAttacher) MountDevice(spec *volume.Spec, devicePath string, deviceMountPath string) error {
+	klog.Info("vsphere MountDevice", devicePath, deviceMountPath)
 	mounter := attacher.host.GetMounter(vsphereVolumePluginName)
 	notMnt, err := mounter.IsLikelyNotMountPoint(deviceMountPath)
 	if err != nil {
 		if os.IsNotExist(err) {
-			if err := os.MkdirAll(deviceMountPath, 0750); err != nil {
-				klog.Errorf("Failed to create directory at %#v. err: %s", deviceMountPath, err)
+			dir := deviceMountPath
+			if runtime.GOOS == "windows" {
+				dir = filepath.Dir(deviceMountPath)
```

Comment:
later if error occurs for `diskMounter.FormatAndMount`
we do `os.Remove(deviceMountPath)`, may be for windows you need `os.Remove(filepath.Dir(deviceMountPath))`

Reply:
There will be a symlink at `deviceMountPath`, I think we want to delete that, the same as on Linux. It looks like on both platforms the directory is left?